plugins/influxdb/database_credentials.go

```diff
@@ -0,0 +1,40 @@
+package influxdb
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.DatabaseCredentials,
+		DocsURL: sdk.URL("https://docs.influxdata.com/influxdb/v2.7/reference/cli/influx/config/create/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Host,
+				MarkdownDescription: "InfluxDB host name",
+			},
+			{
+				Name:                fieldname.Organization,
+				MarkdownDescription: "InfluxDB Organization name",
+			},
+			{
+				Name:                fieldname.AccessToken,
+				MarkdownDescription: "InfluxDB Token value",
+				Secret:              true,
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer:           importer.TryEnvVarPair(defaultEnvVarMapping),
```

Comment:
Could we also import the secrets from the configuration file stored at `~/.influxdbv2/configs`? 

Docs: https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/config/

I think I saw something about OS-specific storage path a few days ago but I cannot find it at the moment. Would you know of anything like that, @bala-ceg?

Reply:
Sorry i didn't remember seeing anything like this, while building this plugin

Reply:
We'd still like to import secrets from the configuration file though. In the spirit of moving this PR forward, I'll approve this PR but I am creating a new issue to handle the config file importer: 

- https://github.com/1Password/shell-plugins/issues/355